store/job_storer.go

```diff
@@ -0,0 +1,43 @@
+package store
+
+import (
+	"context"
+	models "github.com/ONSdigital/dp-search-reindex-api/models"
+	"github.com/ONSdigital/log.go/log"
+	"time"
+)
+
+type JobStorer struct {
+	JobsMap map[string]models.Job
+}
+
+// CreateJob creates a new Job resource and stores it in the JobsMap
+func (js *JobStorer) CreateJob(ctx context.Context, id string) (models.Job, error) {
+
+	log.Event(ctx, "creating job", log.Data{"id": id})
+	newJob := models.Job{
+		ID:          id,
+		LastUpdated: time.Now().UTC(),
+		Links: &models.JobLinks{
+			Tasks: "http://localhost:12150/jobs/" + id + "/tasks",
+			Self:  "http://localhost:12150/jobs/" + id,
+		},
+		NumberOfTasks:                0,
+		ReindexCompleted:             time.Time{}.UTC(),
+		ReindexFailed:                time.Time{}.UTC(),
+		ReindexStarted:               time.Time{}.UTC(),
```

Comment:
Use  `time.Now().UTC()` instead.

Each of these time fields will be different (admitted only by nanoseconds) - I'm not sure if this matters or not but consider creating a single time var before you create the struct and using the var for each of the time fields.

Reply:
time.Time{}.UTC() gives zero time (as the reindex is neither started, failed, nor completed when the job details are first created) but I have made a variable, as suggested, for clarity and to avoid repeating code